streaming/src/main/scala/org/apache/spark/streaming/dstream/DStreamCheckpointData.scala

```diff
@@ -87,9 +87,12 @@ class DStreamCheckpointData[T: ClassTag](dstream: DStream[T])
               if (fileSystem == null) {
                 fileSystem = path.getFileSystem(dstream.ssc.sparkContext.hadoopConfiguration)
               }
-              fileSystem.delete(path, true)
+              if (fileSystem.delete(path, true)) {
+                logInfo("Deleted checkpoint file '" + file + "' for time " + time)
```

Comment:
While you're welcome to use string interpolation here while you're at it, this seems fine